src/lib.rs

```diff
@@ -428,7 +427,10 @@ fn iter_config_paths(font_dir: &Path) -> Result<impl Iterator<Item = PathBuf>, C
         else {
             return false;
         };
-        stem.starts_with("config") && (extension == "yaml" || extension == "yml")
+        !stem.starts_with("fontbakery")
+            && !stem.starts_with(".")
+            && !stem.starts_with("METADATA")
```

Comment:
note: This was included to filter out https://github.com/djrrb/Bungee/blob/master/METADATA.yml